vms/platformvm/txs/txheap/by_end_time.go

```diff
@@ -22,11 +22,11 @@ func NewByEndTime() TimedHeap {
 }
 
 func (h *byEndTime) Less(i, j int) bool {
-	iTime := h.txs[i].tx.Unsigned.(txs.Staker).EndTime()
-	jTime := h.txs[j].tx.Unsigned.(txs.Staker).EndTime()
+	iTime := h.txs[i].tx.Unsigned.(txs.PreContinuousStakingStaker).EndTime()
```

Comment:
I believe this whole package will go as soon as continuous fork gets activated and we clenaup the mempool.